bundles/statistics/statsgrid2016/FlyoutManager.js

```diff
@@ -73,7 +74,20 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.FlyoutManager', function (insta
         if (!flyout) {
             return;
         }
-        flyout.move(flyout.options.pos.x, flyout.options.pos.y, true);
+
+        const indicators = this.service.getStateService().getIndicators();
```

Comment:
And you can remove getStateService() from here if you assign it to this.service on the constructor.

Reply:
This change is now included to PR.